modules/src/ics23_commitment/merkle.rs

```diff
@@ -1,5 +1,6 @@
 use crate::ics23_commitment::commitment::CommitmentPrefix;
-use ibc_proto::commitment::MerklePath;
+
+use ibc_proto::ibc::commitment::MerklePath;
```

Comment:
Seems we always import types from ibc proto as `use ibc_proto::ibc::X`, so I'm wondering why is the `ibc` namespace in the middle necessary. We could just say `use ibc_proto::X`.

Reply:
See this PR for a (poorly explained) rationale: https://github.com/informalsystems/ibc-proto/pull/9

Reply:
Also, the ibc_proto crate exports other types which are not ibc-specific so imho it makes sense to qualify their usage, though we could te-export them at the top level but that may cause similar issues to the one mentioned in the PR linked above.